packages/flutter_tools/lib/src/android/gradle_utils.dart

```diff
@@ -54,14 +54,14 @@ const String oneMajorVersionHigherJavaVersion = '23';
 //
 // Supported here means supported by the tooling for
 // flutter analyze --suggestions and does not imply broader flutter support.
-const String maxKnownAndSupportedGradleVersion = '8.7';
+const String maxKnownAndSupportedGradleVersion = '8.10';
```

Comment:
Are point releases significant here? I.e. will using `8.10.2` result in some warning?

Reply:
Whoops! That was on the wrong branch, should be on https://github.com/flutter/flutter/pull/157032. Thanks for pointing it out!

Reply:
I think the answer to the question by the way is that yes they are significant, and I'll want to figure out how we handle patch versions for that PR. 

I suspect that the correct way to handle them is to basically make the max known and supported `8.10.99` for example, because I don't think we will want to update that value for each patch version, and it's probably fine to just automatically assume new patch versions for a known minor version are supported.

But that is a question for that other PR!